apps/bynder/src/index.js

```diff
@@ -237,6 +238,16 @@ setup({
       default: validAssetTypes.join(','),
       required: true,
     },
+    {
+      id: 'compactViewMode',
+      name: 'Compact View Mode',
+      type: 'List',
+      value: 'MultiSelect,SingleSelect,SingleSelectFile',
+      default: 'MultiSelect',
+      description:
+        'Set the Compact View to allow multiple or single asset selection at a time. "SingleSelectFile" is the only mode that allows a specific derivative to be selected. See https://developer-docs.bynder.com/ui-components for more info on Modes.',
```

Comment:
When using SingleSelect or SingleSelectFile you can also select multiple assets by just opening the dialog again. I would've expected that view mode would replace the existing asset. Seems like a bug to me.